updatecli/updatecli.d/update-plugins.yaml

```diff
@@ -0,0 +1,143 @@
+---
+scms:
+  github:
+    kind: github
+    spec:
+      user: "{{ .github.user }}"
+      email: "{{ .github.email }}"
+      owner: "{{ .github.owner }}"
+      repository: "{{ .github.repository }}"
+      token: "{{ requiredEnv .github.token }}"
+      username: "{{ .github.username }}"
+      branch: "{{ .github.branch }}"
+
+sources:
+  jenkins2.289.x:
```

Comment:
Looks like this file needs to be maintained when adding or removing lines. Would you mind adding a note to https://github.com/jenkinsci/bom/#lts-lines to this effect?